wasm/notebook/webpack.config.js

```diff
@@ -29,7 +29,23 @@ module.exports = (env = {}) => {
                 {
                     test: /\.css$/,
                     use: [MiniCssExtractPlugin.loader, 'css-loader']
-                }
+                },
+                {
+
+                    test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,           
```

Comment:
Does this do anything? It doesn't look like you're importing any fonts, just the ones from google fonts

Reply:
Will check and update

Reply:
updated to 
```
{
                    test: /\.(woff(2)?|ttf)$/,
                    use: {
                        loader:"file-loader",
                        options: { name: "fonts/[name].[ext]" }
                    },
                }
```

I need this to import fonts for katex. The files are `.woff`, `.woff2` and `.ttf` and go under `/fonts`

![Screenshot from 2020-09-26 11-09-43](https://user-images.githubusercontent.com/521705/94343908-7a261380-ffe9-11ea-8c5e-8f0944cdc245.png)

Is there a better way to do this in the webpack config?

I didn't know katex used 60 fonts for math... :open_mouth:  should I look into why and force maybe a smaller amount? (since this is an example, it might be ok -- but happy to look into it).